README.md

```diff
@@ -14,16 +14,16 @@ It is the default face of the Material Design Specification, and available to ev
 
 In 2016 the OpenType format was [updated](https://medium.com/@tiro/https-medium-com-tiro-introducing-opentype-variable-fonts-12ba6cd2369) with a revised version of Apple's “TrueType GX” run-time interpolation technology.
 The primary motivation for the OpenType specification authors to do this was obtaining dramatic reductions in the file-size of large font families, such as Google's [Noto](https://github.com/googlei18n/noto-fonts/tree/master/phaseIII_only/unhinted/variable-ttf) fonts. 
-The Google Fonts team intends to go beyond compression: To make Variable Fonts meaningful.
+The Google Fonts team intends to go beyond compression: They want to make variable fonts *meaningful*.
 
-Variations technology offers the potential of new ways to do typography, and could improve Material Specification Typography guidance in myriad ways.
+Variable fonts create a new dimension in typographic possibilities, and could improve Material Specification Typography guidance in myriad ways.
 Today the Material typography guidelines include adjustments for tracking and font weight that are used to improve typographic-color and texture.
 Variations allow those adjustments to be built-in and “just work.” 
 
 After the public announcement of the technology, the Google Fonts team commissioned Font Bureau to develop a pilot project, [Amstelvar](https://github.com/TypeNetwork/Amstelvar), that demonstrates a novel approach that uses a system of inter-related axes. 
 This increases the fine typographic control available to typographers, and enables a new kind of “parametric” typography.
 
-Learn more about this new typography at [variablefonts.typenetwork.com](https://variablefonts.typenetwork.com)
```

Comment:
I think this is correct; "new typography" is a reference to the "New Typography" movement (eg https://www.moma.org/calendar/exhibitions/1013) and that website is about the ways typography can be done with variable fonts.